### PR TITLE
Header Hierarchy Jump on Catalog Import Page

### PR DESCRIPTION
**Pull request by Matth-Ng**
## Hey, I just made a Pull Request!

On the catalog-import page, I've updated the hierarchy order for accessibility standards. I've added a header to the `infocard` title to be `<h3>` and changed the text section headers to be `<h4>`

import component new

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by Matth-Ng** [comment]
This fix is for issue https://github.com/backstage/backstage/issues/11620

File: .changeset/violet-apples-repair.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-import': minor
```
Comment (Rugvip) [inline .changeset/violet-apples-repair.md:2]: Would call this a `patch`

**Review by Rugvip — COMMENTED** [review]
Hmm, makes sense from an a11y point of view, but it does look a bit shite doesn't it? 😅  Really I think out header stylings are to blame here though.

Does it look better if we shift things over in some direction? Thinking `h2` + `h3`, or `h4` + `h5`, etc.

**Comment by Matth-Ng** [comment]
@Rugvip, I agree. Thanks to some help I've found that I can update the headers while keeping the styling. I've updated the code to match 
import component new old styling
the old original styling

**Review by jhaals — APPROVED** [review]
Looking good, thanks 🎉

**Comment by suuus** [comment]
Congrats on your first PR merged @Matth-Ng! Happy to count you as a contributor.

**Pull request merged**